src/internal/m365/collection/groups/backup.go

```diff
@@ -78,7 +79,8 @@ func CreateCollections[C graph.GetIDer, I groupsItemer](
 		cdps[scope.Category().PathType()],
 		bpc.Options,
 		counter,
-		errs)
+		errs,
+		useLazyReader)
```

Comment:
nit: could you please put this above the options parameter? I know not all code follows the standard, but I think we're sort of trying to keep options, counters, and the fault bus as the last few parameters where possible